src/fabric_cicd/fabric_workspace.py

```diff
@@ -323,65 +322,65 @@ def _convert_path_to_id(self, item_type, path):
         # if not found
         return None
```

Comment:
For future - we can consider moving these methods into the Item class as they are item object specific.